chroma_core/models/filesystem.py

```diff
@@ -299,6 +302,9 @@ def description(self):
     def get_deps(self):
         deps = []
         for t in ObjectCache.get_targets_by_filesystem(self.filesystem_id):
+            # Report filesystem available if MDTs other than 0 are unmounted
+            if t.target_type().lower() == "mdt" and not t.get_label().ends_with("MDT0000"):
```

Comment:
This logic looks to be the same, would it be worth abstracting it? Also, i'm not sure the `continue` is necessary. Maybe something like:
```
def target_is_mdt0(t):
    return t.target_type().lower() == "mdt" and t.get_label().ends_with("MDT0000")
```
Then:
```
if target_is_mdt0(t):
    deps.append(DependOn(t, "mounted", fix_state="unavailable"))
```

Reply:
by the way, i'm ok with it the way it is, just thought this might be a bit cleaner than using a continue.